src/main/resources/react/src/pages/UserInfo/components/buttonsAddRemoveChild/blockButtons.js

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
+
+import ButtonsAddRemoveChild from './buttonsAddRemoveChild';
+
+
+const ButtonsBlock = styled.div`
+	display: flex;
+  justify-content: flex-end;
+  margin-bottom: ${ props => props.mb };
+`;
+
+const AddRemoveDiv = styled.div`
+	width: 415px;
+	display: flex;
+	justify-content: space-between;
+`;
+
+function BlockButtons( { mb, index, id, add, remove } ) {
+
+	const itemAdd = useCallback( () => {
+		add();
+	}, [ add ] );
+
+	const itemRemove = useCallback( () => {
+		remove( id );
+	}, [ remove, id ] );
+
+	return (
+		<ButtonsBlock mb={ mb }>
+			<AddRemoveDiv>
+				<ButtonsAddRemoveChild
+					text={ 'Добавить поле' }
+					addRemove={ 'add' }
+					click={ itemAdd }
+				/>
+				{ Boolean( index ) && (
+					<ButtonsAddRemoveChild
```

Comment:
I see here duplication of logic. You could refactor it and to create new function that would get `text`, `removeText` and `click` and return `<ButtonsAddRemoveChild ...`